.github/workflows/oclif-release.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '16.x'
+          node-version: '18.x'
```

Comment:
We have changed to node v16 because flex plugin does not support node v18. It will break flex plugin.